DevelopGitHub/GitHubExtension/GitHubExtension.Security.WebApi.Library/Controllers/RepositoryController.cs

```diff
@@ -70,6 +70,7 @@ public class RepositoryController : BaseApiController
         [Authorize]
         [Route("api/repos/current")]
         [HttpPatch]
+        //TODO: use DTO
         public async Task<IHttpActionResult> UpdateProject(Repository repo)
```

Comment:
This method was written by my colleague. I only fixed it to save and update claims right. Ok I'll add Dto